x-pack/elastic-agent/pkg/core/plugin/app/monitoring/beats/beats_monitor.go

```diff
@@ -35,7 +41,7 @@ func NewMonitor(downloadConfig *artifact.Config) *Monitor {
 	return &Monitor{
 		operatingSystem: downloadConfig.OS(),
 		installPath:     downloadConfig.InstallPath,
-		config:          &monitoringConfig.MonitoringConfig{},
+		config:          &defaultMonitoringConfig,
```

Comment:
As this introduces defaults, I wonder if we still need it to be a reference on line 35?